src/shared/util.js

```diff
@@ -64,8 +64,10 @@ const AnnotationEditorType = {
 const AnnotationEditorParamsType = {
   FREETEXT_SIZE: 0,
   FREETEXT_COLOR: 1,
-  INK_COLOR: 2,
-  INK_THICKNESS: 3,
+  FREETEXT_OPACITY: 2,
+  INK_COLOR: 3,
+  INK_THICKNESS: 4,
+  INK_OPACITY: 5,
```

Comment:
*Given that editing still is in development, and not officially released, this isn't an issue now; however it may become one in the future.*

To avoid issues, let's group them like this instead to leave room for future additions without requiring backwards-incompatible changes. (Here I'm *assuming* that `10` parameters for each Annotation-type is plenty.)
```suggestion
  FREETEXT_SIZE: 1,
  FREETEXT_COLOR: 2,
  FREETEXT_OPACITY: 3,
  INK_COLOR: 11,
  INK_THICKNESS: 12,
  INK_OPACITY: 13,
```